test/Parallel/traversals/gdistances.jl

```diff
@@ -0,0 +1,19 @@
+@testset "Parallel.BFS" begin
+
+
+    g5 = SimpleDiGraph(4)
+    add_edge!(g5, 1, 2); add_edge!(g5, 2, 3); add_edge!(g5, 1, 3); add_edge!(g5, 3, 4)
+    
+    for g in testdigraphs(g5)
+        @test @inferred(Parallel.gdistances(g, 1))  == dists(shortest_paths(g, 1))
```

Comment:
Aren't `dists` and `shortests_paths` now deprecated?

Reply:
`dists` is; `shortest_paths` is definitely not. That's the new API, and it defaults to BFS.